packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -61,24 +62,32 @@ const SideNavCollapsedContainer = styled("div", {
 
 const SideNavExpandContainer = styled("div", {
   shouldForwardProp: (prop) =>
-    prop !== "odysseyDesignTokens" && prop !== "isSideNavCollapsed",
+    prop !== "odysseyDesignTokens" &&
+    prop !== "isSideNavCollapsed" &&
+    prop !== "expandedWidth",
 })(
   ({
     odysseyDesignTokens,
     isSideNavCollapsed,
+    expandedWidth,
   }: {
     odysseyDesignTokens: DesignTokens;
     isSideNavCollapsed: boolean;
+    expandedWidth: string;
   }) => ({
     backgroundColor: odysseyDesignTokens.HueNeutralWhite,
     flexDirection: "column",
     display: "flex",
     opacity: isSideNavCollapsed ? 0 : 1,
     visibility: isSideNavCollapsed ? "hidden" : "visible",
-    width: isSideNavCollapsed ? "0" : "100%",
+    width: isSideNavCollapsed ? "0" : expandedWidth,
     transitionProperty: "opacity, width",
     transitionDuration: odysseyDesignTokens.TransitionDurationMain,
     transitionTimingFunction: odysseyDesignTokens.TransitionTimingMain,
+    borderStyle: odysseyDesignTokens.BorderStyleMain,
```

Comment:
Guessing this could be simplified to 
```css
borderRight: odysseyDesignTokens.BorderWidthMain odysseyDesignTokens.BorderStyleMain odysseyDesignTokens.HueNeutral50
```
And probably don't need the `borderWidth: 0` declaration